express/services/players.js

```diff
@@ -1,7 +1,11 @@
 const models = require('../../sequelize/models');
 
-function getAll() {
-    return models.player.findAll();
+function getAll({sortField = 'id', sortDirection = 'ASC'}) {
```

Comment:
same with spaces here, please set this property in code styles so It was applied automatically